app/views/contact_information/edit.html.erb

```diff
@@ -15,7 +15,7 @@
         autofocus: true,
         options: { maxlength: 10 },
         notes: "No dashes or spaces (example: 5555555555)." %>
-      <%= f.mb_input_field :email, "What is your email address?", notes: ["If you don’t have one or you don’t want to receive emails from us you can leave this blank."] %>
+      <%= f.mb_input_field :email, "What is your email address?<span class='form-card__optional'>(optional)</span>", notes: ["If you don’t have one or you don’t want to receive emails from us you can leave this blank."] %>
```

Comment:
to simplify this, we can add an `optional` flag to our `MbFormBuilder` that adds this markup. Let's pair on that after retro!